incubator/etcd/README.md

```diff
@@ -49,6 +49,7 @@ The following table lists the configurable parameters of the etcd chart and thei
 | `affinity`              | affinity settings for pod assignment | `{}`                                               |
 | `nodeSelector`          | Node labels for pod assignment       | `{}`                                               |
 | `tolerations`           | Toleration labels for pod assignment | `[]`                                               |
+| `env`                   | Optional environment variables       | `[]`                                               |
```

Comment:
conventionally these have been named "extraEnv" - could you do the same? also if you use a yaml copy structure instead of name/value pairs it is possible to refer to things like the downwardApi etc (more flexible)